.github/workflows/ci_pr_to_jira.yml

```diff
@@ -0,0 +1,61 @@
+name: PR to Jira. Flaky
+on:
+   workflow_run:
+     workflows: [Pull Request Open]
+     types: [completed]
+jobs:
+   download:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+        - name: Install xmlstarlet tool
```

Comment:
It's a minor thing, but can we add a space between each entry in the `steps` array? IMO it really helps readability, especially when we have longer `run` or `script` elements in a step.